client/css/style.css

```diff
@@ -1109,7 +1109,7 @@ kbd {
 #chat .quit .content,
 #chat .topic .content,
 #chat .topic_set_by .content {
-	color: #999;
+	color: #444;
```

Comment:
This is like `time`: it doesn't add a lot of value, and too dark it would clutter the place pretty hard. Even #777 is too dark, IMO, but I would be okay with `#777` since they will be much less cluttered after #759.